### ISSUE TITLE
Cannot tab through social buttons on share card

### ISSUE THREAD

**Issue by 6a68** [mozilla-services/screenshots#4668]
Currently, tabbing into the visible share card (on shot or my shots page) jumps over the social buttons, only landing on the share URL and the 'copy' button for the URL.

**Comment by punamdahiya** [comment]
We should fix this issue with #4776 , new shot page UI copy button update

**Comment by JackBlack85** [comment]
2018-08-21 19:28 GMT+02:00, punamdahiya <notifications@github.com>:

**Comment by chenba** [comment]
This is a duplicate of #4412.
